tools/tune/tdfb/bf_design.m

```diff
@@ -119,6 +123,10 @@
 			+(bf.mic_y(n) - bf.mic_y(m))^2 ...
 			+(bf.mic_z(n) - bf.mic_z(m))^2);
             Gamma_vv(:,n,m) = sinc(2*pi*f*lnm/bf.c);
+		if isequal(DEBUG_DIAG_LOAD_ON,1)
```

Comment:
if DEBUG_DIAG_LOAD_ON == 1 would work too. Though I think the proposed diagonal load could be made default without option. Or make it an bf_defaults.m set option if you think enabled/disabled would make sense.

Reply:
Sure, I will make it as default